apollo/uma/queries.ts

```diff
@@ -7,6 +7,9 @@ export const ACTIVE_POSITIONS = gql`
       positions(first: 1000, where: { collateral_gt: 0 }) {
```

Comment:
when the graph is deployed on Kovan we can validate the liquidation structure using:
```
liquidations {
          id
        }
```